pkg/console/operator/operator.go

```diff
@@ -144,23 +143,23 @@ func (c *consoleOperator) Key() (metav1.Object, error) {
 
 func (c *consoleOperator) Sync(obj metav1.Object) error {
 	startTime := time.Now()
-	logrus.Infof("started syncing operator %q (%v)", obj.GetName(), startTime)
-	defer logrus.Infof("finished syncing operator %q (%v) \n\n", obj.GetName(), time.Since(startTime))
+	klog.V(4).Infof("started syncing operator %q (%v)", obj.GetName(), startTime)
+	defer klog.V(4).Infof("finished syncing operator %q (%v) \n\n", obj.GetName(), time.Since(startTime))
 
 	// we need to cast the operator config
 	operatorConfig := obj.(*operatorsv1.Console)
 
 	// ensure we have top level console config
 	consoleConfig, err := c.consoleConfigClient.Get(api.ConfigResourceName, metav1.GetOptions{})
 	if err != nil {
-		logrus.Errorf("console config error: %v \n", err)
+		klog.Errorf("console config error: %v \n", err)
```

Comment:
Shouldn't be neccesary to add newline here as well. Check https://github.com/kubernetes/klog/blob/master/klog.go#L1261-L1283